### PR TITLE
[Feature] The sharing url changes to the same as original report url

### PR DESCRIPTION
**Pull request by popcornylu**
The sharing url changes to the same as original report url

**PR checklist**

- [x] Ensure you have added or ran the appropriate tests for your PR.
- [x] DCO signed

**What type of PR is this?**
Feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
sc-30593

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

### CODE REVIEW

**Comment by popcornylu** [comment]
Fix sc-30771

File: piperider_cli/cloud/__init__.py [hunk 1]
```diff
@@ -181,15 +182,18 @@ def get_default_project(self) -> PipeRiderProject:
...
-                # Legacy projects api response
-                if project.get('is_default'):
-                    return PipeRiderProject(project)
-            else:
-                # New projects api response
-                for p in project.get('projects', [])[:1]:
-                    return PipeRiderProject(p)
+        responseData = response.json()
+
+        def genProjects():
```
Comment (kentwelcome) [inline piperider_cli/cloud/__init__.py:187]: Is it a typo of `getProjects`?

**Review by kentwelcome — APPROVED** [review]
LGTM. Just raise a possible typo minor issue.

File: piperider_cli/cloud/__init__.py (hunk 1 above)
Comment (popcornylu) [reply piperider_cli/cloud/__init__.py:187]: It is generator, so it means "GENerate projects"

**Pull request merged**